test/index.js

```diff
@@ -1091,6 +1065,114 @@ describe('PubSub', function() {
     });
   });
 
+  describe('getClient_', function() {
+    var CONFIG = {
+      client: 'SubscriberClient',
+      reqOpts: {},
+      gaxOpts: {},
+    };
+
+    beforeEach(function() {
+      pubsub.config = CONFIG;
+    });
+
+    it('should cache the client', function(done) {
+      pubsub.getClient_(CONFIG, function(err, client1) {
+        assert.strictEqual(null, err);
+        pubsub.getClient_(CONFIG, function(err, client2) {
+          assert.strictEqual(null, err);
+          assert.strictEqual(client1, client2);
+          done();
+        });
+      });
+    });
+  });
+
+  describe('request', function() {
```

Comment:
Most of the tests in this block are actually tests for `getClient_`. I'll point out the one that is correct.

Reply:
Sure - there was no getClient_ block before :) I will move them accordingly.